helm_deploy/values-dev.yaml

```diff
@@ -11,18 +11,16 @@ image:
 
 ingress:
   enabled: true
-  enable_whitelist: true
   host: hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk
   path: /
 
 env:
-  API_ENDPOINT_URL: https://api-dev.prison.service.justice.gov.uk/
-  OAUTH_ENDPOINT_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth/
+  HMPPS_AUTH_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth/
   HMPPS_COOKIE_NAME: hmpps-session-dev
   REMOTE_AUTH_STRATEGY: "true"
   WEB_SESSION_TIMEOUT_IN_MINUTES: "60"
   VIDEO_LINK_ENABLED_FOR: "WWI"
-  TOKENVERIFICATION_API_URL: https://token-verification-api-dev.prison.service.justice.gov.uk
-  TOKENVERIFICATION_API_ENABLED: true
+  TOKEN_VERIFICATION_API_URL: https://token-verification-api-dev.prison.service.justice.gov.uk
+  TOKEN_VERIFICATION_API_ENABLED: true
   REDIS_ENABLED: true
   SUPPORT_URL: https://support-dev.hmpps.service.justice.gov.uk/
```

Comment:
I couldn't find `HMPPS_COOKIE_NAME`, `REMOTE_AUTH_STRATEGY`, `VIDEO_LINK_ENABLED_FOR` and `SUPPORT_URL` in the codebase. Could we remove those too?

Reply:
can do